source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2452,7 +2460,234 @@ export class MediaSource extends SerializableObject {
     }
 }
 
+export abstract class MediaPlayer {
```

Comment:
we should update the schema to account for `MediaSources` changes

Reply:
Will do

Reply:
Done. I had to introduce schema 1.6 which wasn't there even though v1.6 and Carousel were already introduced. Someone will have to add the schema for Carousel.